src/identity/types.ts

```diff
@@ -174,6 +174,7 @@ export type PrimaryDerivedKeyInfo = IdentityDerivePayload & {
 export type StoredUser = {
   publicKey: string;
   primaryDerivedKey: PrimaryDerivedKeyInfo;
+  derivedKeyRegistered?: boolean;
```

Comment:
can we make this property part of the PrimaryDerivedKeyInfo instead of a top level StoredUser param since it is actually related to the primaryDerivedKey. I originally created this `primaryDerivedKey` field with the vague notion that people might later want to introduce other derived keys or have "purpose" based derived keys like openfund. I don't know that it would ever happen but it feels better to have the registered flag there.

Reply:
yep - I'll move that in there as well.